### PR TITLE
[NON-MODULAR] Remove piratespeak from the language menu

### PR DESCRIPTION
**Pull request by SarmentiCampbell**
## About The Pull Request
 Piratespeak has been removed from the language menu.

## How This Contributes To The Skyrat Roleplay Experience
Afaik both piratespeak and monkey are gameplay language and i was told to remove it, so might aswell but since peoples want monkie too much, only piratespeak got removed

## Changelog

:cl:
balance: station crew is no longer able to speak  piratespeak.
/:cl:

### CODE REVIEW

**Comment by desertmote** [comment]
Keep monkey in or face my wrath.

**Comment by tf-4** [comment]
On my request

**Comment by azrael-x86** [comment]
desouling

**Comment by desertmote** [comment]
monkey is not a meme language or a game impacting language. if people are taking it as “meme” then finally crack down on the amount of languages we can have. we have TWO WHOLE BONUSSES. i dont even fill those on most my characters.

**Comment by Iamgoofball** [comment]
keep monkey in, this breaks my character even though i can't play them

**Comment by Iamgoofball** [comment]
if you want to claim that monkey and/or piratespeak is a "meme" so are bright neon glow in the dark wolf anthros but nobody's out here calling them a meme and calling for their removal, because that'd be dumb and gatekeeping

restricting people from being creative because something is "overdone" is hypocrisy at minimum and absurd gatekeeping at worst

**Comment by Iamgoofball** [comment]
additionally, piratespeak is accessible via the crew anyways via wearing a pirate hat from the autodrobe, of which there are several

im less attached to that and would have removed it for being an antag language but since it was already a crew accessible language i figured what the hell, might as well

**Comment by desertmote** [comment]
> keep monkey in, this breaks my character 

ditto

**Comment by Iamgoofball** [comment]
in retrospect, someone brought up that piratespeak's accessibility via the hat requires you to forego space proofing, which i agree with as a balance idea

:+1: on removing pirate, :-1: on removing monkey

**Comment by Jezzared** [comment]
We still have curators to translate piratespeak anyhow. If pirates are speaking over public comms crew will understand them anyways, and if they've got a pirate's headset the pirates already have bigger issues.

**Review by Zonespace27 — APPROVED** [review]
I approve the pirate removal and I don't care if monkey language exists or stays

File: code/modules/language/piratespeak.dm [hunk 1]
```diff
@@ -10,3 +10,4 @@
 	"shiver","timbers","matey","swashbuckler"
 	)
 	icon_state = "pirate"
+	secret = TRUE // SKYRAT EDIT
```
Comment (GoldenAlpharex) [inline code/modules/language/piratespeak.dm]: Just add that in a modular file, by just doing this:
```
/datum/language/piratespeak
    secret = true```

I don't remember by heart what kind of files we already have for languages, but you should modularize this, there's no reason it's not modular.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**